lib/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+  end
+end
```

Comment:
Well done! You create Namable class with a method 'correct_name' that raises an error 🔥